lib/srv/discovery/fetchers/aws-sync/s3.go

```diff
@@ -107,8 +109,22 @@ func (a *awsFetcher) fetchS3Buckets(ctx context.Context) ([]*accessgraphv1alpha.
 			if err != nil {
 				collect(nil, trace.Wrap(err, "failed to fetch bucket %q acls policies", aws.ToString(bucket.Name)))
 			}
+
+			tagsOutput, err := s3Client.GetBucketTaggingWithContext(ctx, &s3.GetBucketTaggingInput{
```

Comment:
Should we test this?

Reply:
Added in [f0dcf3c](https://github.com/gravitational/teleport/pull/45364/commits/f0dcf3c5fd17085baeff4f0444fac8156d160bab)